.github/scripts/rust/setup.py

```diff
@@ -24,10 +24,11 @@
 # rust-toolchain.toml
 TOOLCHAINS = {
     "packages/libs/error-stack": ["1.63", "beta-2022-09-25"],
+    "packages/libs/deer": ["1.63", "beta-2022-09-25"]
```

Comment:
Why we are testing `beta` for this crate as well? For `error-stack`, that beta is required to test backtraces on the stable channel, for `deer` this shouldn't matter.

Reply:
Great, thanks!